From 2378ef39c5a0ef17e92ebc44bf368661d8d8d463 Mon Sep 17 00:00:00 2001
From: Alan Conway <aconway@apache.org>
Date: Mon, 6 Jun 2011 15:54:14 +0000
Subject: [PATCH 6/6] Bug 711081 - qpid-cluster -s <ID> crashes with
 "'NoneType' object has no attribute 'split'"

NO-JIRA: Fix typo in qpid/tools/src/py/qpid-cluster.

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@1132683 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit c77fd59988abef185c4a19a71a8646b8ff176605)
---
 qpid/tools/src/py/qpid-cluster |    9 +++++----
 1 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/qpid/tools/src/py/qpid-cluster b/qpid/tools/src/py/qpid-cluster
index 312d59f..d4f9391 100755
--- a/qpid/tools/src/py/qpid-cluster
+++ b/qpid/tools/src/py/qpid-cluster
@@ -247,7 +247,7 @@ def main(argv=None):
         parser.add_option("-k", "--all-stop", action="store_true", default=False, help="Shut down the whole cluster")
         parser.add_option("-f", "--force", action="store_true", default=False, help="Suppress the 'are you sure' prompt")
         parser.add_option("-n", "--numeric", action="store_true", default=False, help="Don't resolve names")
-	
+
         opts, args = parser.parse_args(args=argv)
 
         if args:
@@ -268,12 +268,12 @@ def main(argv=None):
 
         if opts.del_connection:
             config._delConn = opts.del_connection
-            if len(config._delConn.split(":")) != 2: 
+            if len(config._delConn.split(":")) != 2:
                 parser.error("Member ID must be of form: <host or ip>:<number>")
 
         if opts.stop:
-            config._stopID = opts.stop
-            if len(config._stopId.split(":")) != 2: 
+            config._stopId = opts.stop
+            if len(config._stopId.split(":")) != 2:
                 parser.error("Member ID must be of form: <host or ip>:<number>")
 
         config._stopAll = opts.all_stop
@@ -303,6 +303,7 @@ def main(argv=None):
 
         bm.Disconnect()
     except Exception, e:
+        raise
         print str(e)
         return 1
 
-- 
1.7.4.4

